modules/security-context-constraints-psa-alert-eval.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * authentication/understanding-and-managing-pod-security-admission.adoc
+
+:_content-type: PROCEDURE
+[id="security-context-constraints-psa-alert-eval_{context}"]
+= Identifying pod security violations
+
+The `PodSecurityViolation` alert does not provide details on which workloads are causing pod security violations. You can identify the affected workloads by reviewing the Kubernetes API server audit logs. This procedure uses `must-gather` to gather the audit logs and then searches for the `pod-security.kubernetes.io/audit-violations` annotation.
+
+.Prerequisites
+
+* You have installed `jq`.
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. To gather the audit logs, enter the following command:
++
+[source,terminal]
+----
+$ oc adm must-gather -- /usr/bin/gather_audit_logs
+----
+
+. To output the affected workload details, enter the following command:
++
+[source,terminal]
+----
+$ zgrep -h pod-security.kubernetes.io/audit-violations must-gather.local.<archive_id>/quay*/audit_logs/kube-apiserver/*log.gz \
+  | jq -r 'select((.annotations["pod-security.kubernetes.io/audit-violations"] != null) and (.objectRef.resource=="pods")) | .objectRef.namespace + " " + .objectRef.name + " " + .objectRef.resource' \
+  | sort | uniq -c
+----
+Replace `must-gather.local.<archive_id>` with the actual directory name.
```

Comment:
Doesn't matter for the rendered version, but could add a `+` to avoid some of the funny highlighting in source.

Reply:
I don't see the funny highlighting (maybe editor-dependent?), but you're right that a + here is right. I'm actually surprised it didn't throw it off :thinking: